tests/test_bot.py

```diff
@@ -503,7 +506,47 @@ def test_send_message(self):
                 channel='channel_id',
                 as_user=True,
                 attachments=json.dumps(json.loads(expected_attachments)),
-                text=None)
+                text=None,
+                user=None)
+
+
+class TestSendEphemeralMessage(TestPhialBot):
+    '''Test phial's send_message function when sending an ephemeral message'''
+
+    def test_send_message(self):
+        self.bot.slack_client = MagicMock()
+        message = Response(channel="channel_id",
+                           ephemeral=True,
+                           text="Test text",
+                           user="user_id")
+        self.bot.send_message(message)
+
+        self.bot.slack_client.api_call.assert_called_with(
+                'chat.postEphemeral',
+                channel='channel_id',
+                as_user=True,
+                attachments='null',
+                text='Test text',
+                user='user_id')
+
+
+class TestSendMessageDefaultsEphemeralToFalse(TestPhialBot):
+    '''Test phial's send_message function does not send ephemeral
+       messages by default'''
+
+    def test_send_message(self):
```

Comment:
Tests around the same functionality should be in one class. Can this be renamed to something like `test_ephemeral_defaults_to_false` and be moved into the `TestSendEphemeralMessage` class.